k8s/service_controller/model/service_instance.go

```diff
@@ -22,9 +22,9 @@ type LastOperation struct {
 }
 
 type CreateServiceInstanceRequest struct {
```

Comment:
Yeah, I think we should clean this model up a bit with things like schemas in there, etc. I'd rather do the clean up of the model in a separate PR, since it will have ramifications to the entire stack.